contracts/api3-server-v1/proxies/InverseApi3ReaderProxyV1.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.27;
+
+import "./adapters/BaseApi3AggregatorAdapter.sol";
+import "./interfaces/IInverseApi3ReaderProxyV1.sol";
+
+/// @title An immutable proxy contract that inverts the value returned by an
+/// IApi3ReaderProxy data feed
+/// @dev This contract implements the AggregatorV2V3Interface to be compatible
+/// with Chainlink aggregators. This allows the contract to be used as a drop-in
+/// replacement for Chainlink aggregators in existing dApps.
+/// Refer to https://github.com/api3dao/migrate-from-chainlink-to-api3 for more
+/// information about the Chainlink interface implementation.
+contract InverseApi3ReaderProxyV1 is
+    BaseApi3AggregatorAdapter,
+    IInverseApi3ReaderProxyV1
+{
+    /// @notice IApi3ReaderProxy contract address
+    address public immutable override proxy;
+
+    /// @param proxy_ IApi3ReaderProxy contract address
+    constructor(address proxy_) {
+        if (proxy_ == address(0)) {
+            revert ZeroProxyAddress();
+        }
+        proxy = proxy_;
+    }
+
+    /// @notice Returns the inverted value of the underlying IApi3ReaderProxy
+    /// @dev This inverts the 18-decimal fixed-point value using 1e36 / value.
+    /// It is assumed that operation might overflow if `baseValue` is too small
+    /// and will revert on division by zero. No safety checks are performed to
```

Comment:
I'm thinking it would make more sense to revert after all. It seems better to not report a price rather then report an invalid one. The former is easier for consumers to protect against and they may have better protections in place.

Reply:
https://github.com/api3dao/contracts/pull/472/commits/398d1f3bdf2fd82775fd437165ea4a79b02560f7